pkg/controller/utils/clusterdeployment.go

```diff
@@ -23,17 +23,21 @@ func IsFakeCluster(cd *hivev1.ClusterDeployment) bool {
 	return fakeCluster && err == nil
 }
 
-// IsClusterPausedOrRelocating checks if the syncing to the cluster is paused or if the cluster is relocating
-func IsClusterPausedOrRelocating(cd *hivev1.ClusterDeployment, logger log.FieldLogger) bool {
+// IsClusterPaused checks if the syncing to the cluster is paused
+func IsClusterPaused(cd *hivev1.ClusterDeployment, logger log.FieldLogger) bool {
 	if paused, err := strconv.ParseBool(cd.Annotations[constants.SyncsetPauseAnnotation]); err == nil && paused {
 		logger.WithField("annotation", constants.SyncsetPauseAnnotation).Warn("syncing to cluster is disabled by annotation")
 		return true
 	}
+	return false
+}
+
+// IsClusterRelocating checks if the cluster is relocating
+func IsClusterRelocating(cd *hivev1.ClusterDeployment, logger log.FieldLogger) bool {
```

Comment:
If the above is correct, the rest of this refactor can be reverted back to original form.